packages/im-components/src/atoms/tabs/Tabs.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import PropTypes from "prop-types";
+// import { Tabs as AntdTabs } from "antd";
+import { Tabs } from "antd";
+import { EMPTY_STRING } from "imbase/constants/base.constants"
+
+// const Tabs = (props) => {
+//     const { className, type, size, restProps } = props;
+//     return (
+//         <AntdTabs
+//             {...restProps}
+//             className={className}
+//             type={type}
+//             size={size}
```

Comment:
Remove Comments, if not for test purpose

Reply:
Done